PermissionsScrapper/PermissionsScrapper.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30104.148
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PermissionsScrapper", "PermissionsScrapper\PermissionsScrapper.csproj", "{C1727BFF-DB1A-496C-A615-A6786C2A6591}"
```

Comment:
should this really by Scrapper or Scaper?

Reply:
It is scraper. We aren't scrapping permissions.

Reply:
Renamed appropriately